gitlab/createGitLabBotUser.sh

```diff
@@ -15,112 +15,124 @@ set -o nounset
 set -o pipefail
 
 IFS=$'\n\t'
-script_name="$(basename ${0})"
-script_folder="$(dirname $(readlink -f "${0}"))"
+SCRIPT_NAME="$(basename ${0})"
+SCRIPT_FOLDER="$(dirname $(readlink -f "${0}"))"
 
-export password_store_dir=~/.password-store/cbi-pass
-gitlab_pass_domain="gitlab.eclipse.org"
+if [[ ! -f "${SCRIPT_FOLDER}/../.localconfig" ]]; then
+  echo "ERROR: File '$(readlink -f "${SCRIPT_FOLDER}/../.localconfig")' does not exists"
+  echo "Create one to configure the location of the password store. Example:"
+  echo '{"password-store": {"cbi-dir": "~/.password-store/cbi"}}'
+fi
+PASSWORD_STORE_DIR="$(jq -r '.["password-store"]["cbi-dir"]' "${SCRIPT_FOLDER}/../.localconfig")"
+PASSWORD_STORE_DIR="$(readlink -f "${PASSWORD_STORE_DIR/#~\//${HOME}/}")"
+export PASSWORD_STORE_DIR
 
+GITLAB_PASS_DOMAIN="gitlab.eclipse.org"
 
-personal_access_token=$(cat ../.localconfig | jq -r '."gitlab-token"')
-token_header="PRIVATE-TOKEN: ${personal_access_token}"
-api_base_url="https://gitlab.eclipse.org/api/v4"
+PERSONAL_ACCESS_TOKEN="$(cat ../.localconfig | jq -r '."gitlab-token"')"
```

Comment:
Should be `${SCRIPT_FOLDER}/../.localconfig`

Reply:
Fixed